application/src/main/resources/db/V5__Add_Warn_System.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE warn_system
+(
+    /**
+     * This was taken from https://github.com/YusufsDiscordbot/Yusuf-s-Moderation-Bot and changed.
+     */
```

Comment:
Remove this comment, it is not in compliance with our CLA - we own the code, not you.